hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieWriteHandle.java

```diff
@@ -141,6 +141,13 @@ private static Schema getWriteSchema(HoodieWriteConfig config) {
     return new Schema.Parser().parse(config.getWriteSchema());
   }
 
+  /**
+   * Whether to include the '_hoodie_operation' field in the metadata fields.
+   */
+  protected boolean withOperationField() {
```

Comment:
why can't this be just a `WriteConfig`? that we can set to different values for spark and flink